2-mandatory/1-create-functions.js

```diff
@@ -24,18 +38,30 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(arr) {
+  return arr.map((str) => str.trim().replace("/", "").toLowerCase());
 }
-
+// console.log(tidyUpString([
+//     "/Daniel",
+//     " /Sanyia",
+//     "AnTHonY",
+//     "irina",
+//     " Gordon",
+//     "ashleigh   ",
+//     "   Alastair  ",
+//     " anne marie  ",
+//   ]))
 /*
 Write a function that:
 - Takes an array and an index as input.
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, index) {
+  return arr.filter((v) => arr.indexOf(v) !== index);
```

Comment:
This is good. How many arguments does the "callback function" that you pass into `Array.filter` get given?